lib/mock_suey/sorbet_rspec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+
+# Let methods with sig receive double/instance_double arguments
+T::Configuration.call_validation_error_handler = lambda do |signature, opts|
```

Comment:
It reminds me this RSpec Sorbet integration: https://github.com/samuelgiles/rspec-sorbet/blob/4d0e6479453b3b4ef36d2d6a2df8282ae559368b/lib/rspec/sorbet/doubles.rb#L104

Are we doing something similar here? Maybe, we can rely on the `rspec-sorbet` gem? (That's something we can consider in the future)

Reply:
It uses a similar approach and calls the same method `T::Configuration.call_validation_error_handler` from Sorbet.

The first problem I see here is that they may not work together at the moment. The reason for that is this line:
`T::Configuration.send(:call_validation_error_handler_default, signature, opts)`
Most probably, it will not call the handler defined in the gem [rspec-sorbet](https://github.com/samuelgiles/rspec-sorbet/blob/4d0e6479453b3b4ef36d2d6a2df8282ae559368b/lib/rspec/sorbet/doubles.rb#L16 )

I will try to reproduce the error and write a comment about the result below. Anyway, it should be easy to fix. RSpec Sorbet uses a correct approach to override this method.


It should be possible to integrate RSpec Sorbet gem if this is needed I guess.

Reply:
I fixed this issue in another branch: [link to the commit](https://github.com/iurev/mock-suey/commit/1d4a6e018df59c1e846561fb44bd70d562ed30f7)
I will add this change to the `master` branch if you don't mind.